frontend/server/src/Controllers/Course.php

```diff
@@ -3908,6 +3926,12 @@ private static function getCommonCourseDetails(
             'school_id' => intval($course->school_id),
             'school_name' => null,
             'start_time' => $course->start_time,
+            'languages' => !is_null(
+                $course->languages
+            ) ? explode(
+                ',',
+                $course->languages
+            ) : [],
```

Comment:
```suggestion
            ) : null,
```

recuerda que `null` es "no hay filtro: todos los lenguajes se permiten" y `[]` es "sí hay filtro: ningún lenguaje es permitido"

Reply:
Listo